ui/app/AppLayouts/Communities/panels/SharedAddressesAccountSelector.qml

```diff
@@ -153,7 +153,6 @@ StatusListView {
                     symbol: d.collectiblesNamesToSymbols[firstEntry.name] || "",
                     enabledNetworkBalance: e.length,
                     account: firstEntry.ownership[0].accountAddress,
-                    type: Constants.TokenType.ERC721,
```

Comment:
Not needed any longer?

Reply:
Now the `type` role is added at the end by `ConcatModel`, so there is no need to add it here.